ocaml/gencert/lib.ml

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
i needed this code from xapi, so am exposing it as a library from gencert

i don't think this is right, perhaps it belongs in the network client library instead?

Reply:
Does `ocaml/xapi/helpers.ml > get_management_ip_addr` not work in this case? (It retrieves the ip from the inventory)

Even `ocaml/xapi/xapi_mgmt_iface.ml` uses this function.

Reply:
I see that you've changed _after_ this commit; it's clear now that this deals with ipv6 as well and the function I mention gets changed to use this one as well.

Reply:
i think i'll just get rid of the commit 'share get_management_ip_logic'

i.e. I'll merge it into the previous